week05_explore/week5.ipynb

```diff
@@ -173,9 +173,9 @@
     "\n",
     "The bandit has $K$ actions. Action produce 1.0 reward $r$ with probability $0 \\le \\theta_k \\le 1$ which is unknown to agent, but fixed over time. Agent's objective is to minimize regret over fixed number $T$ of action selections:\n",
     "\n",
-    "$$\\rho = T\\theta^* - \\sum_{t=1}^T r_t$$\n",
+    "$$\\rho = T\\theta^* - \\sum_{t=1}^T \\theta_{a_t}$$\n",
     "\n",
-    "Where $\\theta^* = \\max_k\\{\\theta_k\\}$\n",
+    "Where $\\theta^* = \\max_k\\{\\theta_k\\}$ and $\\theta_{a_t}$ corresponds to the chosen action $a_t$ on each step.\n",
```

Comment:
This is not true. The agent only ever observes reward samples, not probabilities themselves.

Reply:
Thank you for the reply! Let me try to provide more details regarding my initial point.
I completely agree with you that the agent still observes only reward samples, but its goal is to minimise the total regret defined as the sum of the following terms: **expected** reward of the optimal action minus **expected** reward of the chosen action (see slide 8 in [week5](https://docviewer.yandex.com/view/484004083/?page=8&*=dueTMLM9L6F9Q78cv8PSVHPbokt7InVybCI6InlhLWRpc2stcHVibGljOi8vK0IyeEVVVmdPVzlPakxqU08vNTNONGZIQUVaWU1oU3RNYUM5QlBaVE1kND0iLCJ0aXRsZSI6ImxlY3R1cmU1X3NwcmluZzIwMTgucGRmIiwibm9pZnJhbWUiOmZhbHNlLCJ1aWQiOiI0ODQwMDQwODMiLCJ0cyI6MTY3MjE2MTE1NDM0MiwieXUiOiI5NjI4NjgyNTE2NTA3ODU5ODQifQ%3D%3D)). 
That is the reason why for Bernoulli bandit we are using the maximum probability of 1 as the first term and why I suggest using the probability of 1 for the chosen action (not just reward's sample) as the second term.
Please see page 68 in [TS_tutorial](https://web.stanford.edu/~bvr/pubs/TS_Tutorial.pdf) which is attached inside this notebook.
However, we still do not know neither optimal probability of 1 nor probability of 1 for the chosen action and, thus, do not use them for the optimisation.